main.c

```diff
@@ -411,7 +411,10 @@ static const char usage[] =
 	"  -b #rrggbbaa Set background color.\n"
 	"  -c #rrggbbaa Set border color.\n"
 	"  -s #rrggbbaa Set selection color.\n"
-	"  -w n         Set border weight.\n";
+	"  -w n         Set border weight.\n"
+	"  -f s         Set output format.\n";
+
+static char format[128] = "%x,%y %wx%h";
```

Comment:
This should be a local variable in `main`, and can be `strdup`'ed so that we don't have a length limit.

Reply:
Not sure why you'd use `strdup` here. Instead, I chose to initialise `format` to `NULL`, point it to `optarg` if the format is specified, and if not, set it to the default format.

Reply:
Oh, right, that should work :+1: